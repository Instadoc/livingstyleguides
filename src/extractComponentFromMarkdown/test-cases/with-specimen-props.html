<p>Specimen with <code>hidden</code> prop:</p>
<pre><code class="language-specimen-1.html">&#x3C;b>Specimen 1&#x3C;/b>
</code></pre>
<pre><code class="language-specimen-1.css">b { color: red }
</code></pre>
<pre><code class="language-specimen-1.js">var foo = 'not hidden'
</code></pre>
<pre><code class="language-specimen-1.js">var bar = 'hidden'
</code></pre>
<p>Specimen with frontmatter props:</p>
<pre><code class="language-specimen-2.html">&#x3C;b>Specimen 2&#x3C;/b>
</code></pre>
<pre><code class="language-specimen-2.css">b { color: green }
</code></pre>