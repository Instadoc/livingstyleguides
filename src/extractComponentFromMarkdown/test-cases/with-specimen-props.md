---
name: Component Name
category: Component Category
---

Specimen with frontmatter props:

```specimen.html
---
key: value
---
<b>Specimen</b>
```

```specimen.css hidden
---
key: value
list:
- one
- two
- three
---
b { color: green }
```
