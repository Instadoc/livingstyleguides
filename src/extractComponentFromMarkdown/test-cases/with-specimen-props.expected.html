<p>Specimen with frontmatter props:</p>
<pre><code class="language-specimen.html">&#x3C;b>Specimen&#x3C;/b>
</code></pre>